__tests__/app.test.js

```diff
@@ -103,9 +104,44 @@ describe("GET", () => {
                     expect(typeof article.votes).toBe("number")
                     expect(typeof article.article_img_url).toBe("string")
                     expect(article.hasOwnProperty("body")).toBe(false)
-                    expect(articles).toBeSortedBy("created_at", {descending: true})
                 })
             })
         })
     })
+    describe("/api/article/:article_id/comments", () => {
+        test("200 - returns all comments for an article", () => {
+            return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({body: {comments}}) => {
+                expect(comments).toBeSortedBy("created_at", {descending: true})
+                comments.forEach(comment => {
```

Comment:
this also needs a length check to prevent a false positive, see my previous comments